Dockerfile

```
@@ -62,9 +62,14 @@ FROM --platform=$TARGETOS/$TARGETARCH localhost:5000/base-php:$TARGETARCH AS fin
 
 WORKDIR /var/www/html
 
-# Install additional required libraries
+# Install additional required libraries and runtime dependencies for plugins
 RUN apk add --no-cache \
-    caddy ca-certificates supervisor supercronic fcgi
+    caddy ca-certificates supervisor supercronic fcgi \
+    nodejs npm yarn bash 7zip bzip2-dev
```

```suggestion
    nodejs npm yarn bash p7zip bzip2-dev
```
As far as I know this Image is based of of Alphine and the `7zip` package is `p7zip`
https://pkgs.alpinelinux.org/package/v3.16/main/armv7/p7zip

---

I see alpine 3.16. That is dead old.